### DISCUSSION TITLE
cells are removed

### DISCUSSION THREAD

**Discussion by sinahidd3n** [UvaCsl/HemoCell#17] [General discussion]
Hello,

Is there any possible way to disable the "cell deletion" or "cell removing" functions due to excessive force?
I need the cells to continue high deformations during the simulation.

Thanks

**Comment by gzavo** [comment]
No, HemoCell removes them around the threshold of numerical instability. If you keep them in, the simulation will fail.
If you need to compute high forces the way to go is decreasing the time-step size.
Practically you exchange higher computational cost (since the simulations will take longer) for more numerical stability.
